docs/Gemfile

```diff
@@ -17,8 +17,12 @@
 
 source "https://rubygems.org"
 
+# Keep these specifications as flexible as possible and leave it to Bundler
+# to pin versions in the lock file.
+# To update the lock file, run `bundle update`.
+# Version constraint reference: https://guides.rubygems.org/patterns/#declaring-dependencies
 gem "ffi", "1.15.5"
-gem "jekyll", "4.3.2"
+gem "jekyll", "~> 4.3"
```

Comment:
I would like to migrate all of the dependency specifications here to use the [pessimistic operator][1], but I leave that to a future PR.

[1]: https://thoughtbot.com/blog/rubys-pessimistic-operator